src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/InstanceEntryPointTableNode.cs

```diff
@@ -50,46 +50,45 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
             Dictionary<byte[], BlobVertex> uniqueFixups = new Dictionary<byte[], BlobVertex>(ByteArrayComparer.Instance);
             Dictionary<byte[], BlobVertex> uniqueSignatures = new Dictionary<byte[], BlobVertex>(ByteArrayComparer.Instance);
 
-            foreach (MethodWithGCInfo method in factory.EnumerateCompiledMethods())
+            foreach (MethodWithGCInfo method in factory.EnumerateCompiledMethods(null, true))
```

Comment:
Nit:

```suggestion
            foreach (MethodWithGCInfo method in factory.EnumerateCompiledMethods(null, genericInstantiations: true))
```

This is a rather weird API and I'm quite sure nobody is going to remember what these parameters mean.

Reply:
That's very true. I'll update the names of the parameters and make all the callers use them to try and add some self-documenting here.